docs/mine/lotus/split-markets-miners.md

```diff
@@ -122,29 +173,45 @@ lotus-miner --markets-repo=~/.lotusmarkets init service --type=markets \
                                                         /tmp/backup.cbor
 ```
 
-3. Optionally update your miner's `multiaddr` on-chain - in case your `markets` instance is publicly exposed at a different location compared to your existing monolith miner, you also need to update your `multiaddr` on-chain and advertise the correct one to clients:
+3. Optionally update your miner's `multiaddr` on-chain - in case your `markets`
+instance is publicly exposed at a different location compared to your existing
+monolith miner, you also need to update your `multiaddr` on-chain to advertise
+the correct address to clients:
 
 ```shell
 lotus-miner actor set-addrs <NEW_MULTIADDR>
 ```
 
-### 4. Move the DAG store directory to the markets node repository
+#### 4. Move the DAG store directory to the markets node repository
+
+If you are running a lotus version with a DAG store, you can optionally move the
+DAG store directory to the lotus markets repository, to avoid having to reindex
+all storage deals when the markets node starts up.
 
 ```shell
 mv ~/.lotusminer/dagStore ~/.lotusmarkets/
 ```
 
-### 5. Start the `mining/sealing/proving` miner process without the markets subsystem
+#### 5. Start the `mining/sealing/proving` miner process without the markets subsystem
+
+1. Update your `config.toml` to set the `EnableMarkets` option to `false`.
 
-1. Update your `config.toml` and set `EnableMarkets` option to `false`.
+2. Start the `mining/sealing/proving` node (with the default LOTUS_MINER_PATH, which
+should point to your `mining/sealing/proving` node repo).
```

Comment:
During all this time the miner process was down? If that's the case, that's potentially significant downtime. We should either put a big warning at the top, or rejig the instructions so that they don't require so much downtime (e.g. we can change configurations in-place and simply restart the miner?)

Reply:
I think in practice you can just bounce your miner at this point - I believe the only time it needs to be down is when you do the backup. Would need to verify that with Anton though.

Reply:
Hmm actually the miner needs to be *up* when you do the backup. But at the same time you don't want to be accepting any deals as that would change state. So maybe the user should change their miner config first (so they disable the markets process), then bring the miner up, do the backup, and then bring up the markets node.

Reply:
Makes sense. I would expect us to stop accepting deals, then backup, then stand up the new markets node and start accepting deals again. Otherwise the migration is not transactional and we risk the markets state slipping 👍 

One question: do we have commands to delete the markets state from the `mining/sealing/proving` mode, as it's now rendered irrelevant in its repo?

Reply:
> do we have commands to delete the markets state

I don't think so - and we also copy all that state over to the markets node